tests/validations/mta/mtaValidations.spec.ts

```diff
@@ -94,28 +103,72 @@ describe("mtaValidations", () => {
       await testUpdateMtaDiagnostics([]);
     });
 
+    it("returns no diagnostics when mta.yaml does not exist and clear the current diagnostics", async () => {
```

Comment:
Please add a test where there are new diagnostics too so we can make sure the `undefined` entry is added before the new diagnostics (otherwise the `undefined` will override the new entries)

Reply:
done